src/transformers/__init__.py

```diff
@@ -6871,8 +6873,10 @@
         from .models.idefics2 import (
             Idefics2ForConditionalGeneration,
             Idefics2Model,
+            Idefics2PerceiverResampler,
             Idefics2PreTrainedModel,
             Idefics2Processor,
+            Idefics2VisionTransformer,
```

Comment:
Yes, I wouldn't import it also. I think after the next iteration of changes I'll try to get rid of extra `AutoMap` for sub-configs, this can be removed. That works for most models, but haven't really thought of Idefics